src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -2111,7 +2111,7 @@ public function commerce_get_checkout_url( $checkout_url, $data, $post_id ) {
 	 * @return array
 	 */
 	public function add_cart_url( $urls = [] ) {
-		$cart_url = site_url();
+		$cart_url = site_url( '/' );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 10 spaces but found 1 space

<sub>hash: 11218f44cd335360642c339fe6b09ede</sub>